ops/__init__.py

```diff
@@ -12,31 +12,42 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""The ops library: a Python framework for writing Juju charms.
+"""The API to respond to Juju events and manage the application.
 
-The ops library is a Python framework (`available on PyPI`_) for developing
-and testing Juju charms in a consistent way, using standard Python constructs
-to allow for clean, maintainable, and reusable code.
+This module provides code freatures to your charm, including:
```

Comment:
Small typo: "freatures" should be "features". Sorry I only noticed now!

Reply:
I'll fix this in #1381 to avoid needing a PR just for this little fix.